fecgov/fec-cms
# USA and env-specific  banners

## Summary
- Add Official Gov Website banner 
- Add environment-specific banners with conditional statement in base templates 
- Remove redundant scss imports by refactoring  elections.scss so that is does not import global scss  twice
  -  remove @import `base.scss` 
  - move everything in `base.scss` that's not in `global.scss` into `elections.scss`\
  -  remove stylesheet reference to  the compiled `base.css` in `home_base.html`

**Note:**
Hardcoding the three `img src` paths in the `use-banner.html`  allows us to use one partial for the whole site, rather than having one for Django and one for Jinja. Hardcoding was a way to bypass syntax differences between Django/Jinja for the `static path variable` in templates.

- Resolves #3579 
- Resolves #3678 

## Impacted areas of the application

modified:   data/templates/layouts/main.jinja
modified:   fec/settings/base.py
modified:   fec/static/scss/components/_site-header.scss
modified:   fec/static/scss/elections.scss
modified:   fec/templates/base.html
modified:   fec/templates/home_base.html
new file:    fec/templates/partials/usa-banner.html
new file:    data/templates/partials/data-usa-banner.jinja

## Screenshots
![Apr-18-2020 01-07-19](https://user-images.githubusercontent.com/5572856/79628676-0239f000-8111-11ea-864f-9143b0ecb638.gif)
Moile: 
![Apr-19-2020 22-11-36](https://user-images.githubusercontent.com/5572856/79708239-e3f8ff00-828c-11ea-8948-edf2b4fba2ff.gif)




## Related PRs
https://github.com/fecgov/fec-cms/pull/3611
https://github.com/fecgov/fec-cms/pull/3610

## How to test
 - checkout `feature/3579-3678-usa-and-env-banners`
- `npm run build-sass`
-  `runserver` and review  local env-specific banner and usa-gov banner
- See https://fec-feature-cms.app.cloud.gov to confirm  Feature env-specific banner


____
